demo.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+</head>
+<link rel="stylesheet" href="style.css">
+<body>
+
+    <div class="container">
+        <img style="height: 220px;width: 100%;" src="mountain.jpg" alt="header1.jpg">
+        <div class="main_heading">THIS IS A DEMO WEBSITE</div>
+    </div>
+    <div class="navigation">   
+          <nav>
+            <ul style="list-style-type: none;margin: 0px;padding: 10px;overflow: hidden;background-color: teal;">
```

Comment:
Remove inline css and use external css

Reply:
Inline CSS removed made all the changes and committed with "Inline CSS Removed".